tasks/extract_filepaths_for_granule/schemas/config.json

```diff
@@ -58,7 +58,7 @@
           }
```

Comment:
Could we move fileBucketMaps to this new area, or is that a Cumulus thing? I don't see it in a brief look at their stuff.